Backend/ActiveCruzer/BLL/UserBll.cs

```diff
@@ -38,6 +38,7 @@ public UserBLL(UserManager<User> userManager, IMapper mapper)
             user.Latitude = validatedAddressCoordinates.Latitude;
             user.LastLogin = DateTime.UtcNow;
             user.CreatedOn = DateTime.UtcNow;
+            user.TermsAccepted = DateTime.UtcNow;
```

Comment:
We can not know for sure, that the terms where accepted. The frontend needs to tell us, that the terms where accepted

Reply:
changed to credentials.TermsAccepted